samples/Web/Startup.fs

```diff
@@ -40,7 +40,9 @@ type App = class end
 type Startup() =
     // This method gets called by the runtime. Use this method to add services to the container.
     static member ConfigureServices(services: IServiceCollection, args: ParseResults<Arguments>) : unit =
-        services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Latest) |> ignore
+        services
+            .AddMvc(fun opt -> opt.EnableEndpointRouting <- false)
```

Comment:
`opt.EnableEndpointRouting  <- false` needs to be specified otherwise the sample hit the following runtime error:

```
Application startup exception: System.InvalidOperationException: Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
   at Microsoft.AspNetCore.Builder.MvcApplicationBuilderExtensions.UseMvc(IApplicationBuilder app, Action`1 configureRoutes)
   at Microsoft.AspNetCore.Builder.MvcApplicationBuilderExtensions.UseMvc(IApplicationBuilder app)
   at Web.Startup.Configure(IApplicationBuilder app, IHostEnvironment env) in /Users/k0l01ap/repos/equinox-kim/equinox/samples/Web/Startup.fs:line 92
   at Web.Program.createWebHostBuilder@15-1.Invoke(IApplicationBuilder app) in /Users/k0l01ap/repos/equinox-kim/equinox/samples/Web/Program.fs:line 15
   at Microsoft.AspNetCore.Hosting.WebHostBuilderExtensions.<>c__DisplayClass0_0.<Configure>b__0(WebHostBuilderContext _, IApplicationBuilder app)
   at Microsoft.AspNetCore.Hosting.WebHostBuilderExtensions.<>c__DisplayClass2_1.<Configure>b__2(IApplicationBuilder app)
   at Microsoft.AspNetCore.Hosting.DelegateStartup.Configure(IApplicationBuilder app)
   at Microsoft.AspNetCore.Mvc.Filters.MiddlewareFilterBuilderStartupFilter.<>c__DisplayClass0_0.<Configure>g__MiddlewareFilterBuilder|0(IApplicationBuilder builder)
   at Microsoft.AspNetCore.HostFilteringStartupFilter.<>c__DisplayClass0_0.<Configure>b__0(IApplicationBuilder app)
   at Microsoft.AspNetCore.Hosting.WebHost.BuildApplication()
```

Reply:
Wondering if adding `UseRouting` like https://github.com/jet/dotnet-templates/blob/master/equinox-web-csharp/Web/Startup.cs#L22 can do work instead? (I ran into it and did it that way - I have no fixed opinion on which fix is better but ideally they'd be in sync)

Reply:
Yeah that works, thanks! I'll make the changes.

Reply:
cool, thanks